src/main/java/ratpack/zipkin/internal/RatpackCurrentTraceContext.java

```diff
@@ -23,7 +23,12 @@ public TraceContext get() {
     return registrySupplier.get()
         .maybeGet(TraceContextHolder.class)
         .map(TraceContextHolder::getContext)
-        .orElse(null);
+        .orElseGet(() -> (Execution.isManagedThread())
+          ? Execution.current()
```

Comment:
This does indeed seem redundant.  The tests pass without this so I'm going to remove it.  @edwardsoo-ss is there something else we are overlooking with this?